### PR TITLE
Feature localization

### PR DESCRIPTION
**Pull request by AfsarFresh**
## Users can choose to engage in either English or French in the application.
Verifier_Onboarding

### CODE REVIEW

File: YukonVaccinationVerifier/YukonVaccinationVerifier/Constants/AccessibilityConstants.swift [hunk 1]
```diff
@@ -7,25 +7,72 @@
...
     struct OnBoarding {
-        static let onboardingView = "On Boarding View"
-        static let startScanningButton = "Allow Camera Access" // Constants.Strings.onBoarding.buttonTitle
-        static let title = "BC Vaccine Card Verifier" // Constants.Strings.onBoarding.title
-        static let subtitle = "Businesses can scan official digital or paper BC vaccine cards" // Constants.Strings.onBoarding.subtitle
-        static let phoneImage = "Phone Image"
+        static var onboardingView: String {
+            LanguageService.dynamicLocalizedString("Accessibility.Label.onboardingView") }
+        static var startScanningButton: String {
+            LanguageService.dynamicLocalizedString("Accessibility.Label.allowCameraAccess") }
```
Comment (freshfawzy) [inline YukonVaccinationVerifier/YukonVaccinationVerifier/Constants/AccessibilityConstants.swift]: please align block curly braces  or be inline

File: YukonVaccinationVerifier/YukonVaccinationVerifier/Constants/AccessibilityConstants.swift [hunk 2]
```diff
@@ -7,25 +7,72 @@
...
+        static var title: String { .yukonVaccinationVerifier }
+        static var subtitle: String { .useThis_QRcode }
+        static var phoneImage: String {
+            LanguageService.dynamicLocalizedString("Accessibility.Label.phoneImage") }
```
Comment (freshfawzy) [inline YukonVaccinationVerifier/YukonVaccinationVerifier/Constants/AccessibilityConstants.swift]: same here

File: YukonVaccinationVerifier/YukonVaccinationVerifier/OnBoarding/OnBoardingView.swift [hunk 3]
```diff
@@ -7,11 +7,13 @@
...
 import UIKit
 
-class OnBoardingView: UIView {
+/// Shows basic info of the app and asks for Camera permisions
+internal final class OnBoardingView: UIView {
     
     var buttonCallback: (()->Void)?
 
     // MARK: Outlets
+    @IBOutlet weak var phoneImgVw: UIImageView!
```
Comment (freshfawzy) [inline YukonVaccinationVerifier/YukonVaccinationVerifier/OnBoarding/OnBoardingView.swift:16]: what about renaming it; "phoneImageView"? more readable I guess

**Review by freshfawzy — COMMENTED** [review]
Great work Afsar

File: YukonVaccinationVerifier/YukonVaccinationVerifier/OnBoarding/OnBoardingView.swift (hunk 3 above)
Comment (AfsarFresh) [reply YukonVaccinationVerifier/YukonVaccinationVerifier/OnBoarding/OnBoardingView.swift:16]: Agreed Fawzy.
But I would vouch for shorter names as long as they convey the right element.

**Review by freshfawzy — APPROVED** [review]

**Comment by AfsarFresh** [comment]
> Great work Afsar

Thanks, Fawzy, for the review. It was helpful.

**Pull request merged**